### PR TITLE
refactor: correct formatting, make small improvements

### PR DESCRIPTION
**Pull request by j9t**

### CODE REVIEW

**Comment by j9t** [comment]
@itay1313—please review and challenge my decisions. The update removes unneeded files (per search through the DC), reformats for consistency, adds TODOs for follow-up, makes quotes consistent, and has some small improvements.

**Comment by j9t** [comment]
(To confirm, local tests were pretty basic, but look good.)

**Review by itay1313 — APPROVED** [review]

File: css/customstyles.css [hunk 1]
```diff
@@ -1,1718 +0,0 @@
-body {
```
Comment (itay1313) [inline css/customstyles.css:1]: Overall, good cleaning, but you've removed this file for example. Where did you insert all that code?

File: css/lavish-bootstrap.css [hunk 2]
```diff
@@ -1,6841 +0,0 @@
-/* CSS generated by http://lavishbootstrap.com */
```
Comment (itay1313) [inline css/lavish-bootstrap.css:1]: Same question, you removed this file but I remember that some of those codes have an effect on the website, so where have you inserted the code?

**Review by itay1313 — CHANGES_REQUESTED** [review]

File: css/customstyles.css (hunk 1 above)
Comment (j9t) [reply css/customstyles.css:1]: This style sheet doesn’t seem to be used—nothing in the project points to “customstyles.css” or even “customstyles”. Just double-checked—would this be missing something?

File: css/lavish-bootstrap.css (hunk 2 above)
Comment (j9t) [reply css/lavish-bootstrap.css:1]: lavish-bootstrap.css doesn’t seem to be used—only lavish-bootstrap.min.css (which, ironically, isn’t even minified). To clean up, this update removes the unused lavish-bootstrap.css, and adds a TODO to later rename and minify lavish-bootstrap.min.css.

**Comment by j9t** [comment]
@itay1313, I shared some detail. Please let me know if you have additional information. (I didn’t resolve the comments to give you a chance to review—but if we’re good to go, please resolve and approve. Thanks!)

File: css/lavish-bootstrap.css (hunk 2 above)
Comment (itay1313) [reply css/lavish-bootstrap.css:1]: I will also check this PR to make sure and approve after testing to make sure

**Review by itay1313 — APPROVED** [review]

**Comment by j9t** [comment]
Thanks @itay1313!

**Pull request merged**